gh-136640: Clarify the documentation of the AST module (#136868)
python/cpython: Doc/library/ast.rst

                  end_lineno
                  end_col_offset

>       Most classes part of the AST module have the :attr:`lineno`, :attr:`col_offset`,

Reviewer: ```suggestion
      Most classes in the :mod:`!ast` module have the :attr:`lineno`, :attr:`col_offset`,
```

>       :attr:`end_lineno`, and :attr:`end_col_offset` attributes, including subclasses of

Reviewer: ```suggestion
      :attr:`end_lineno`, and :attr:`end_col_offset` attributes, including all subclasses of
```
